src/handlers/MessageCommandHandler.ts

```diff
@@ -79,29 +113,40 @@ export async function MessageCommandHandler(
 	const client = message.client as GClient;
 
 	const command = Commands.get(commandName);
-	if (!command)
-		return client.options?.unknownCommandMessage ? message.reply({
-			content: (await Util.getResponse('NOT_FOUND', { client })),
-		}) : null;
-
+	if (!command && client.options.unknownCommandMessage)
```

Comment:
same here

Reply:
I have not made it the same as before, but this should work correctly. The reason I changed this was because eslint + prettier was messing things up with tabs+spaces